src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
The tests are failing because the UI that you coded do not follow the expected results. Please come back to the task description to understand the requirements

![Screenshot from 2024-11-25 10-47-17](https://github.com/user-attachments/assets/6fccfbc0-0f6a-417f-b8f6-02edbdf32bdc)